lib/src/compatKtLint0Dot48Dot1/java/com/diffplug/spotless/glue/ktlint/compat/KtLintCompat0Dot48Dot1Adapter.java

```diff
@@ -66,25 +91,18 @@ public String format(final String text, final String name, final boolean isScrip
 
 		EditorConfigOverride editorConfigOverride;
 		if (editorConfigOverrideMap.isEmpty()) {
-			editorConfigOverride = EditorConfigOverride.Companion.getEmptyEditorConfigOverride();
+			editorConfigOverride = EditorConfigOverride.Companion.getEMPTY_EDITOR_CONFIG_OVERRIDE();
 		} else {
 			editorConfigOverride = createEditorConfigOverride(allRuleProviders.stream().map(
 					RuleProvider::createNewRuleInstance).collect(
 							Collectors.toList()),
 					editorConfigOverrideMap);
 		}
 
-		return KtLint.INSTANCE.format(new KtLint.ExperimentalParams(
-				name,
-				text,
-				allRuleProviders,
-				userData,
```

Comment:
Looks like `userData` is not getting passed into `KtLintRuleEngine`. Looks like userData is deprecated in ktlint so not sure if its needed any more.

Reply:
Looks like `userData` is being ignored either way under `0.48.x`